docs/dc-apps-performance-toolkit-user-guide-jira.md

```diff
@@ -882,12 +894,12 @@ To receive scalability benchmark results for four-node Jira DC with app-specific
 1. Run toolkit with docker:
 
    ``` bash
-    cd dc-app-performance-toolkit
-    docker run --shm-size=4g  -v "$PWD:/dc-app-performance-toolkit" atlassian/dcapt jira.yml
+   cd dc-app-performance-toolkit
+   docker run --shm-size=4g --pull="always" -v "$PWD:/dc-app-performance-toolkit" atlassian/dcapt jira.yml
    ```  
 
 {{% note %}}
-Review `results_summary.log` file under artifacts dir location. Make sure that overall status is `OK` before moving to the next steps.
+Review `results_summary.log` file under artifacts dir location. Make sure that overall status is `OK` before moving to the next steps. For enterprise-scale environment run results acceptable success rate for actions is 95% and above.
```

Comment:
How do you think about such a variant as "For an enterprise-scale environment, the acceptable actions success rate is 95% and above." ?

Reply:
Improved wording. Thanks.